From be1f67ab6fde1989af7b2e2946804e9009931a8a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timo=20Ter=C3=A4s?= <timo.teras@iki.fi>
Date: Wed, 3 Dec 2014 16:09:37 +0200
Subject: [PATCH] add arm private syscall numbers

it is part of kernel uapi, and some programs (e.g. nodejs) do use them
---
 arch/arm/bits/syscall.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/bits/syscall.h b/arch/arm/bits/syscall.h
index d1b1c99..df23b83 100644
--- a/arch/arm/bits/syscall.h
+++ b/arch/arm/bits/syscall.h
@@ -340,6 +340,11 @@
 #define __NR_getrandom	384
 #define __NR_memfd_create	385
 
+#define __ARM_NR_breakpoint	0x0f0001
+#define __ARM_NR_cacheflush	0x0f0002
+#define __ARM_NR_usr26		0x0f0003
+#define __ARM_NR_usr32		0x0f0004
+#define __ARM_NR_set_tls	0x0f0005
 
 /* Repeated with SYS_ prefix */
 
-- 
2.2.0

